tools/ci/before_install.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 set -v
+pushd $TRAVIS_BUILD_DIR
```

Comment:
Would it be better to do `pushd $(pwd)`?

Reply:
or even `pushd .`

Reply:
actually I can just do `pushd $PWD` its already in a var